Userland/Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -159,7 +160,9 @@ void ConnectionFromClient::ensure_connection(URL const& url, ::RequestServer::Ca
     if (cache_level == CacheLevel::ResolveOnly) {
         return Core::deferred_invoke([host = url.serialized_host().release_value_but_fixme_should_propagate_errors().to_byte_string()] {
             dbgln("EnsureConnection: DNS-preload for {}", host);
-            (void)gethostbyname(host.characters());
+            ErrorOr<IPv4Address> resolved_host = Core::Socket::resolve_host(host.characters(), Core::Socket::SocketType::Stream);
```

Comment:
```suggestion
            auto resolved_host = Core::Socket::resolve_host(host, Core::Socket::SocketType::Stream);
```

That's still a mouthful, but better.